src/vibe/components/SidebarNav/components/NavSingleItem.js

```diff
@@ -6,9 +6,15 @@ import NavBadge from './NavBadge';
 const NavSingleItem = ({ item }) => {
   const Icon = item.icon && Feather[item.icon] ? Feather[item.icon] : null;
   if (item.external) {
+    let rel = null;
+    
+    if("target" in item && item.target === "_blank") {
+      rel = "noopener noreferrer";
```

Comment:
We can make this a little more concise with a ternary.

`const rel = item.target && item.target === '_blank' ? 'noopener noreferrer' : null;`

What do you think?

Reply:
Agreed, changed it.